Exercise_C/index.css

```diff
@@ -1,4 +1,27 @@
-/* 
-This css sheet is linked to index.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+.box-wrap {
+    display: flex;
+    justify-content: space-between;
+    height: 200px;
+}
+
+.red {
+    background-color: red;
+    align-self: flex-end;
+
+}
```

Comment:
Great use of the containers size as well as align self to move your red box about. 
Superb logic!